sources/RuleEditWindow.cpp

```diff
@@ -22,10 +23,11 @@
 #include "TestView.h"
 
 
-RuleEditWindow::RuleEditWindow(BRect& rect, FilerRule* rule, BHandler* caller)
+RuleEditWindow::RuleEditWindow(FilerRule* rule, BHandler* caller)
 	:
-	BWindow(rect, "Edit rule", B_TITLED_WINDOW,
-		B_ASYNCHRONOUS_CONTROLS | B_NOT_RESIZABLE | B_CLOSE_ON_ESCAPE),
+	BWindow(BRect(0, 0, 400, 0), "Edit rule", B_TITLED_WINDOW,
+		B_ASYNCHRONOUS_CONTROLS | B_NOT_RESIZABLE | B_CLOSE_ON_ESCAPE |
```

Comment:
I think the B_NOT_RESIZABLE should be removed. E.g. a moving action may have a rather long path to the location to move a file. The user may want to resize the window to shoe the complete path.